README.md

```diff
@@ -1,2 +1,8 @@
 # iterativereview
 This repo is to test out Github's iterative review experience. Yay!
+
+This is a line I added.
+
+Here's ANOTHA WUN I added.
+
+This is a line that I will soon delete.
```

Comment:
(Comment on a filtered diff view) (Line deleted/outdated)

Why did you delete this?